system/services/assetManager/ImageMagickWrapper.cfc

```diff
@@ -0,0 +1,173 @@
+component output="false" displayname="ImageMagick"  {
```

Comment:
Could you remove all the `output=false` in the CFC? I realized too late that these are not needed in script based CFCs so there are lots of them in the codebase. But new code should not have `output=false` on script based components or their functions.